free-2.md

```diff
@@ -11,23 +11,49 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met is people who want to figure out what they need to do at any
+moment and what they can do if they are feeling productive. Currently, there is
+some combination of calendar where events can be manually put, potentially todo
+lists, some of which have different types of deadlines which are all mixed
+together. If this language works out, people can have all of their TODO-like
+things all in one place: things such as mandatory meetings presented one way,
+optional events in another, and hard a soft deadlines that come with some
+weighting so the user knows how important they are.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A language is appropriate because the user can write a program that is all of
+their activities. This consolidates all of the relevant information and then
+the program can be executed to tell the user about things they need to do.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I'm really excited by this project because scheduling well sounds like
+something that should be easy but I still have trouble finding something that
+doesn't require me to remember and plan ahead enough. I thought of this idea
+after reading [Michaels' previous project](https://github.com/mculhane/dPete).
+I have avoided reading it in depth to not pollute my ideas, but the idea
+resonances we me a lot.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Person-oriented task management system.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+There will have to be several different ways to interact with the language for
+it to be feasible. There has to be easy ways to add events from emails. When an
+email comes in that says there's an interesting-sounding talk on a certain day,
+it should be very easy to put this in, and change settings like how much you
+want to go to it. There should be some similar interface with TODOs, even for
+things such as "deal with this email." Setting up events outside of email
+should also be easy, and it's important to have a nice interface for recurring
+events.
```

Comment:
How to deal with emails sounds very second-level to me. That is, sort of like the standard library is to Python; the MVP would not include such a thing, but would be directly extendable to it, or capable already of doing it but just not in so few words.

I guess the description of the interface is mostly "similar to TODOs".

What kind of DSL functionality are you thinking of? I think it would be cool if the DSL allowed you to implement your own kinds of functionality. Such as, suppose you didn't support recurring events. But, it was simple enough for a user to make a recipe for such a thing. It could be hard to keep the functionality from getting too powerful though.